src/Apps/Order/Routes/Shipping2/ShippingContext.tsx

```diff
@@ -2,7 +2,7 @@ import { createContext, FC } from "react"
 import {
   ParsedOrderData,
   parseOrderData,
-} from "Apps/Order/Routes/Shipping2/Utils/ShippingContext/parseOrderData"
+} from "Apps/Order/Routes/Shipping2/Utils/parseOrderData"
```

Comment:
moved this into `Utils`, and dropped `ShippingContext` into the route folder route. This parse function is largely a util. If we need to organize this further in the future we can tho, but for now this should be fine.